core/src/main/java/org/commonjava/indy/core/ctl/AdminController.java

```diff
@@ -169,6 +202,16 @@ public void delete( final StoreKey key, final String user, final String changelo
         }
     }
 
+    private void deleteContent( final ArtifactStore store ) throws IndyWorkflowException
+    {
+        List<Transfer> contents = downloadManager.listRecursively( store.getKey(), DownloadManager.ROOT_PATH );
```

Comment:
You should be able to use `contentManager.delete(store, DownloadManager.ROOT_PATH, ...)` I believe.

Reply:
Ok. I will test it.

Reply:
Maybe I miss something? This line is to list all paths in the store. I did not find any code that do cascade deletion from root /  so this code list them and delete them.

Reply:
Deleting the root directory should trigger a cascading deletion. I think there are some details in TransferManagerImpl in Galley...but I'm pretty sure I traced that code out when I was trying to understand the batch-delete code.

Reply:
Yes,TransferManagerImpl can cascade delete from /. I updated the code and ftests passed.